R/utilities-file.R

```diff
@@ -66,7 +66,6 @@ readExcel <- function(path, sheet = NULL, ...) {
 #' @param data Data frame or named list of data frames that will be sheets in
 #' the xlsx
 #' @inheritParams writexl::write_xlsx
-#' @export
 writeExcel <- function(data, path, col_names = TRUE) {
```

Comment:
prepend `.` to the name to stay consistent with naming conventions.

Reply:
done in a2f2994